### ISSUE TITLE
OSX

### ISSUE THREAD

**Issue by RyanRiffle** [jrudio/meteor-electron#13]
I just tested on OSX 10.9.5 and it does not work.
```
Starting Electron...
/bin/sh: /Users/ryan/.meteor/packages/jrudio_electron/.0.0.21.11gjw94++os+web.browser+web.cordova/npm/node_modules/electron-prebuilt/dist/Electron.app/Contents/MacOS/Electron: No such file or directory
```
I'm going to try to look into it a little more.

**EDIT**
Looking into it further shows that `node_modules/electron-prebuilt/dist/` does not contain the electron application `Electron.app`

**Comment by jrudio** [comment]
Hmm, interesting... I just tested it on OSX 10.10.5 it threw an error and restarted the app and booted electron fine, but didn't load the meteor app, just a blank screen.

I installed OS X about a week ago, and I originally written this package when I was on windows. So I'm going to set some time tomorrow to re-write and hopefully there won't be errors!

If you can figure out the issue, though, that would be great!

**Comment by RyanRiffle** [comment]
I restarted meteor as well, but got the same error.

**Comment by jrudio** [comment]
Hey @RyanRiffle, if you are still interested in this, I wrote a new meteor-tool that integrates with meteor. I have only tested it on my machine, but if you want to give it a swing, that'd be great!

In your terminal, navigate to your meteor project:

`meteor --release jrudio:METEOR@1.1.0 add-desktop`

This should take a few minutes to download the necessary meteor stuff and then setup electron for your meteor app.

`meteor --release jrudio:METEOR@1.1.0 run-desktop`

This will startup your meteor app like normal and then when it's ready, it will fire up electron

`meteor --release jrudio:METEOR@1.1.0 build-desktop`

This will package your meteor app using your current machine's info (platform, arch) up into `.electron/out`

And, finally `meteor --release jrudio:METEOR@1.1.0 remove-desktop`

This will remove associated electron files from your project
